File: tools/toos_life/3-MLtools/myTest/test3.py
#!/usr/bin/env python
# -*- coding: UTF-8 -*-
# coding=utf-8 

"""
@author: Li Tian
@contact: litian_cup@163.com
@software: pycharm
@file: test3.py
@time: 2020/5/4 13:08
@desc: 测试专用3：生成数据集
        [make_multilabel_classification详解](https://blog.csdn.net/n889092/article/details/77126754)
"""

from sklearn.datasets import load_breast_cancer
import numpy as np

from MLTools import MLTools
from PreAnalysis import PreAnalysis

data = load_breast_cancer()
X = data['data']
y = data['target']
feature_names = data['feature_names']

# 测试数据分析的热力图
PreAnalysis('/').correlationMatrix(X, names=feature_names)

File: 2-practice_projects/1-java_projects/1-Java从入门到放弃/src/Test3.java
/**
 * @author: Li Tian
 * @contact: 694317828@qq.com
 * @software: pycharm
 * @file: Test3.java
 * @time: 2019/8/20 17:48
 * @desc:
 */

public class Test3 {
    public static void main(String[] args){
        int score = 80;
        String type = score<60? "不及格":"及格";
        System.out.println(type);

        double x = Math.random();
        System.out.println(x);
        System.out.println((int)(6*x + 1));


        outer: for (int i = 101; i < 150; i++) {
            for (int j = 2; j < i / 2; j++) {
                if (i % j == 0){
                    continue outer;
                }
            }
            System.out.print(i + "  ");
        }
    }
}
